public/js/public.js

```diff
@@ -59,12 +66,20 @@ const rotaractCalendarOptions  = {
 			}
 		}
 	}
-}
+};
 
-var calendar;
+let calendar;
```

Comment:
Did you test this one? Not sure if this comes without trouble concerning the scope...

Reply:
Change is motivated by <https://jshint.com/docs/options/#varstmt>. I have not thought about scoping...